packages/js/src/initializers/post-scraper.js

```diff
@@ -134,7 +137,7 @@ export default function initPostScraper( $, store, editorData ) {
 	 * @returns {boolean} True when markers should be shown.
 	 */
 	function displayMarkers() {
-		return ! isBlockEditor() && wpseoScriptData.metabox.show_markers === "1";
+		return ! isBlockEditor() && wpseoScriptData.metabox.show_markers;
```

Comment:
`show_markers` returns a boolean. So checking if `wpseoScriptData.metabox.show_markers === "1"` will never be true.